contracts/native/version-control/src/commands.rs

```diff
@@ -60,12 +65,45 @@ pub fn add_modules(
             // Only owner can add modules
             validate_account_owner(deps.as_ref(), &module.provider, &msg_info.sender)?;
         }
-        MODULE_LIBRARY.save(deps.storage, &module, &mod_ref)?;
+        if config.is_testnet {
+            MODULE_LIBRARY.save(deps.storage, &module, &mod_ref)?;
+        } else {
+            PENDING_MODULES.save(deps.storage, &module, &mod_ref)?;
+        }
     }
 
     Ok(VcResponse::action("add_modules"))
 }
 
+/// Approve or decline a module
+pub fn approve_or_decline_module(
```

Comment:
There is no check to ensure that a module is not in both the approve and reject list, which could cause confusion. 

Also let's stick to one name -> "reject" instead of using "reject" and "decline" interchangeably.